src/SSVOpenHexagon/Data/StyleData.cpp

```diff
@@ -7,8 +7,17 @@
 #include "SSVOpenHexagon/Utils/Match.hpp"
 #include "SSVOpenHexagon/Utils/Color.hpp"
 #include "SSVOpenHexagon/Global/Config.hpp"
+#include "SSVOpenHexagon/Core/HGStatus.hpp"
+#include "SSVOpenHexagon/Utils/Utils.hpp"
 #include "SSVOpenHexagon/Utils/FastVertexVector.hpp"
 
+using namespace std;
+using namespace sf;
+using namespace ssvs;
+using namespace hg::Utils;
+using namespace ssvu;
+using namespace ssvuj;
```

Comment:
This is considered bad practice - namespaces help to understand where the symbols we're using are coming from, especially in a project with many libraries like Open Hexagon.

Reply:
I know, not my thing but Synth's merging purpose idk, told him that you are not using them but kept just in case.